worker/uniter/runner/jujuc/storage-get.go

```diff
@@ -55,23 +57,15 @@ func (c *StorageGetCommand) Run(ctx *cmd.Context) error {
 	if !ok {
 		return nil
 	}
-	values := make(map[string]interface{})
-	var singleValue interface{}
-	for _, key := range c.keys {
-		switch key {
-		case "kind":
-			values[key] = storageInstance.Kind
-		case "location":
-			values[key] = storageInstance.Location
-		default:
-			return errors.Errorf("invalid storage instance key %q", key)
```

Comment:
We just want to ignore invalid keys rather than print an error? I'd prefer an error. Maybe a todo for now?